frontend-ssr/components/cli_git/match_commands.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+class MatchWords extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            value: '',
+            message: '',
+        }
+
+        this.handleInputChange = this.handleInputChange.bind(this);
+        this.handleMatch = this.handleMatch.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+    handleSubmit(event) {
+        event.preventDefault();
+    }
+    handleInputChange(event) {
+        this.setState({
+            value: event.target.value
+        });
+    }
+
+    handleMatch() {
+        let inputValue = this.state.value;
+        let defaultValue = this.props.defaultText;
```

Comment:
Please change to just one prop which can be either an Array or a string. You should check the type of this props and based on it do the rest of the logic.